docs/bigquery-usage.rst

```diff
@@ -315,9 +313,9 @@ Background a query, loading the results into a table:
    """
    >>> dataset = client.dataset('dataset_name')
    >>> table = dataset.table(name='person_ages')
-   >>> job = client.run_async_query(query,
-   ...                              destination=table,
-   ...                              write_disposition='truncate')
+   >>> job = client.run_async_query('fullname-age', query)
+   >>> job.destination_table = table
+   >>> job.write_disposition= 'truncate'
```

Comment:
Jobs must have names (`fullname-age`).  *Except* for the synchronous query job, which doesn't (so I shouldn't be passing it here).